archetype-module-content/src/main/resources/archetype-resources/pom.xml

```diff
@@ -115,27 +115,42 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Spotless plugin -->
             <plugin>
-                <groupId>net.revelc.code</groupId>
-                <artifactId>impsort-maven-plugin</artifactId>
-                <version>1.5.0</version>
-                <dependencies>
-                    <!-- Needed because of maven 3.9.0 backward compatibility issue,
-                         See : https://github.com/revelc/impsort-maven-plugin/issues/64 -->
-                    <dependency>
-                        <groupId>org.codehaus.plexus</groupId>
-                        <artifactId>plexus-utils</artifactId>
-                        <version>3.5.1</version>
-                    </dependency>
-                </dependencies>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.27.2</version>
                 <configuration>
-                    <groups>javax, java, *</groups>
-                    <removeUnused>true</removeUnused>
+                    <java>
+                        <licenseHeader>
+                            <file>${project.basedir}/license-header.txt</file>
+                        </licenseHeader>
+                        <eclipse>
+                            <file>${project.basedir}/OpenMRSFormatter.xml</file>
+                        </eclipse>
+                        <removeUnusedImports/>
+                        <importOrder>
```

Comment:
Don't we want `<removeUnusedImports />` in this block too?

Reply:
we actually don't , ref -> https://github.com/diffplug/spotless/blob/06168eb5ab9e932aee31a0ac2ac25fee67f45475/plugin-maven/src/main/java/com/diffplug/spotless/maven/java/ImportOrder.java#L29.